guides/6-ape/2-limit-order.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: Limit Order (v1)
+title: ApePro Limit Order (v1)
+description: Learn more about Ape Pro's Limit Order mechanism.
+slug: /apepro/limit-order
+---
+
+<head>
+    <title>ApePro Limit Order (v1)</title>
+    <meta name="twitter:card" content="summary" />
+</head>
+
+ApePro Limit Order (v1) allows users to set a Target Price/Mcap (to be referred to interchangeably) for a trade to become automatically be executed once specified conditions are met. Once the order is placed, Ape’s Keeper will continuously monitor the target level and execute the swap when the market price reaches your specified price.
```

Comment:
`to become automatically be executed`, repeated become and be? change to `to be automatically executed`?